neard/src/config.rs

```diff
@@ -368,8 +368,8 @@ impl Genesis {
             gas_price_adjustment_rate: GAS_PRICE_ADJUSTMENT_RATE,
             block_producer_kickout_threshold: BLOCK_PRODUCER_KICKOUT_THRESHOLD,
             validators,
-            developer_reward_percentage: DEVELOPER_PERCENT,
-            protocol_reward_percentage: PROTOCOL_PERCENT,
+            developer_reward_percentage: DEVELOPER_REWARD_RATE,
+            protocol_reward_percentage: PROTOCOL_REWARD_RATE,
```

Comment:
BTW, should this also become `*_rate`? And either way, it is a breaking change -- is this fine?